shell/scripts/test-plugins-build.sh

```diff
@@ -145,4 +148,63 @@ cp ${SHELL_DIR}/list/catalog.cattle.io.clusterrepo.vue ./pkg/test-pkg/list
 FORCE_COLOR=true yarn build-pkg test-pkg | cat
 rm -rf ./pkg/test-pkg
 
+# function to clone repos and install dependencies (including the newly published shell version)
+function clone_repo_test_extension_build() {
+  REPO_NAME=$1
+  PKG_NAME=$2
+
+  echo -e "\nSetting up $REPO_NAME repository locally\n"
+
+  # set registry to default (to install all of the other dependencies)
+  yarn config set registry ${DEFAULT_YARN_REGISTRY}
+
+  rm -rf ${BASE_DIR}/$REPO_NAME
+
+  # cloning repo
+  git clone https://github.com/rancher/$REPO_NAME.git
+  cd ${BASE_DIR}/$REPO_NAME
```

Comment:
Nit: You could use [`pushd`/`popd`](https://www.gnu.org/software/bash/manual/html_node/Directory-Stack-Builtins.html) to manipulate the directory stack, it can be a bit easier to follow where in the stack the shell is located during the script.